Model/Observer/IdentifyFinancingProgram.php

```diff
@@ -23,11 +23,27 @@
 use Magento\Customer\Model\Session;
 use Magento\Framework\App\RequestInterface;
 
+
 /**
  * Identify Financing Program for customer
  */
 class IdentifyFinancingProgram implements ObserverInterface
 {
+     /**
+     * Customer session
+     *
+     * @var \Magento\Customer\Model\Session
+     */
+    protected $customerSession;
```

Comment:
was this part of the changes for this PR?

Reply:
yea, I wanted to create consistency so I remove the "_". 

PHP8.2 also required us to define these values